pkg/api/handlers/libpod/manifests.go

```diff
@@ -162,13 +162,36 @@ func ManifestAdd(w http.ResponseWriter, r *http.Request) {
 	// Wrapper to support 3.x with 4.x libpod
 	query := struct {
 		entities.ManifestAddOptions
-		Images []string
+		Images    []string
+		TLSVerify bool `schema:"tlsVerify"`
 	}{}
 	if err := json.NewDecoder(r.Body).Decode(&query); err != nil {
 		utils.Error(w, http.StatusInternalServerError, errors.Wrap(err, "Decode()"))
 		return
 	}
 
+	authconf, authfile, err := auth.GetCredentials(r)
+	if err != nil {
+		utils.Error(w, http.StatusBadRequest, errors.Wrapf(err, "failed to parse registry header for %s", r.URL.String()))
```

Comment:
`GetCredentials` already adds a pretty much the same “failed to parse registry header” wrapper, so this `Wrapf` seems unnecessary.

(I realize there is exactly one instance of doing that, in this file. It probably slipped through during a cleanup.)